src/helm/benchmark/run_specs.py

```diff
@@ -2243,6 +2285,198 @@ def get_anthropic_hh_rlhf_spec(num_respondents: int, subset: str) -> RunSpec:
+        groups=["decoding_trust", "adv_demonstration"],
+    )
+
+
+@run_spec_function("decodingtrust_ood_robustness")
+def get_ood_robustness_spec(ood_type: str, task: str, demo_name: str, run_id: int, idk: bool) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.decodingtrust_ood_robustness_scenario.OoDRobustnessScenario",
+        args={"ood_type": ood_type, "task": task, "demo_name": demo_name, "run_id": run_id, "idk": idk},
+    )
+    if 0 <= run_id < 3:
+        max_train = 5 if task == "knowledge" else 8
+        adapter_spec = get_few_shot_instruct_adapter_spec(
+            num_outputs=1, max_tokens=16, temperature=0, max_train_instances=max_train
+        )
+    else:
+        adapter_spec = get_instruct_adapter_spec(num_outputs=1, max_tokens=16, temperature=0)
+
+    return RunSpec(
+        name=f"ood_robustness:ood_type={ood_type},task={task},demo_name={demo_name}" + f",run_id={run_id},idk={idk}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_exact_match_metric_specs() if ood_type == "style" else get_ood_knowledge_metric_specs(),
+        groups=["decoding_trust", "ood_robustness"],
+    )
+
+
+@run_spec_function("decodingtrust_fairness")
+def get_fairness_spec(task: str, setting: str) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.decodingtrust_fairness_scenario.FairnessScenario",
+        args={"task": task, "setting": setting},
+    )
+
+    adapter_spec = get_instruct_adapter_spec(num_outputs=1, max_tokens=16, temperature=0)
+
+    return RunSpec(
+        name=f"fairness:task={task}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_fairness_metric_specs() + get_exact_match_metric_specs(),
+        groups=["decoding_trust", "fairness"],
+    )
+
+
+@run_spec_function("decodingtrust_privacy")
+def get_privacy_spec(
+    scenario_name: str,
+    data_file: str,
+    template: str,
+    dataset_size: int,
+    few_shot_num: int,
+    prompt_type: str,
+) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.decodingtrust_privacy_scenario.PrivacyScenario",
+        args={
+            "scenario_name": scenario_name,
+            "data_file": data_file,
+            "template": template,
+            "dataset_size": dataset_size,
+            "few_shot_num": few_shot_num,
+            "prompt_type": prompt_type,
+        },
+    )
+
+    adapter_spec = get_instruct_adapter_spec(num_outputs=1, max_tokens=32, temperature=1)  # temperature>0 for generaion
+
+    return RunSpec(
+        name=f"privacy:scenario_name={scenario_name},template={template},few_shot_num={few_shot_num},prompt_type={prompt_type},dataset_size={dataset_size}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_exact_match_metric_specs(),
```

Comment:
The DecodingTrust privacy metrics seem to be missing.

Reply:
Thanks for the reminder. We added `decodingtrust_privacy_metrics.py`